packages/backend-defaults/src/entrypoints/auth/helpers.test.ts

```diff
@@ -42,6 +42,23 @@ describe('credentials', () => {
       },
     });
 
+    expect(
+      createCredentialsWithUserPrincipal(
+        'user:default/mock',
+        'my-token',
+        undefined,
+        'my-actor',
+      ),
+    ).toEqual({
+      $$type: '@backstage/BackstageCredentials',
+      version: 'v1',
+      principal: {
+        type: 'user',
+        userEntityRef: 'user:default/mock',
+        actor: { type: 'service', subject: 'my-actor' },
+      },
+    });
```

Comment:
this is unrelated; i just noticed that this test didn't have a case for the actor input